packages/client/package.json

```diff
@@ -21,6 +21,7 @@
     "react": "^17.0.1",
     "react-art": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-draggable": "^4.4.3",
```

Comment:
From a look at their readme it looks really nice, and maybe they have solved the problem of mouse leaving component?

Reply:
I tried it and it worked nicely locally by just encapsulating the stickynote with a <Draggable/> component.

I was however not sure how to make the Draggable-component play together with our data store. It's probably possible to implement via the Draggable-props so will investigate further since it would fix the note being pushed around.

Reply:
Yes, judging by the [Controlled vs Uncontrolled](https://github.com/react-grid-layout/react-draggable#controlled-vs-uncontrolled) section, we want to pass the `position` as prop to draggable according to our `props.note.positio`, and then call `moveNote` in handlers for `onDrag` and `onStop`.